### DISCUSSION TITLE
Queue multiple onload events sync

### DISCUSSION THREAD

**Discussion by shawnwall** [bigskysoftware/htmx#2834] [Q&A]
I've tried to find an answer to this, and seen things such as [this question](https://github.com/bigskysoftware/htmx/discussions/687), but could not figure out via code... My goal here is to sequentially load n different elements on load. Contrived/simplified code:

```
<div hx-get="/path1" hx-trigger="load"></div>
<div hx-get="/path2" hx-trigger="load"></div>
<div hx-get="/path3" hx-trigger="load"></div>
```

I'd like them all to use a single `queue`, perhaps via `hx-sync`. I've tried to do this via a parent element, etc, but could not figure out a solution so far. Thanks~

----

Update, this is the closest I got so far, but its only executing 2 of the 4 requests:

```
<div class="charts">
<div hx-get="/path1" hx-trigger="load" hx-sync=".charts:queue last"></div>
<div hx-get="/path2" hx-trigger="load" hx-sync=".charts:queue last"></div>
<div hx-get="/path3" hx-trigger="load" hx-sync=".charts:queue last"></div>
<div hx-get="/path4" hx-trigger="load" hx-sync=".charts:queue last"></div>
</div>
```

I tried various options for the secondary `queue` argument.

This also did NOT work, and everything was issued at once:

```
<div hx-get="/chart" hx-trigger="load from:body"></div>
<div hx-get="/chart2" hx-trigger="load from:body"></div>
<div hx-get="/chart3" hx-trigger="load from:body"></div>
```

**Comment by shawnwall** [comment]
ok, i guess this isnt possible?

**Comment by blurrcat** [comment]
What about only rendering the load element for path 1 during initial render, and then return element to load path2 in the response of path1, in addition to your regular response?
